docs/reference/builder.md

```diff
@@ -203,43 +210,34 @@ The following is an example `.dockerignore` file:
     */*/temp*
     temp?
     *.md
-    !LICENSE.md
+    !README*.md
+    README-secret.md
 ```
 
 This file causes the following build behavior:
 
 | Rule           | Behavior                                                                                                                                                                     |
 |----------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `*/temp*`      | Exclude all files with names starting with`temp` in any subdirectory below the root directory. For example, a file named`/somedir/temporary.txt` is ignored.                 |
+| `*/temp*`      | Exclude all files with names starting with`temp` in any subdirectory immediately below the root directory. For example, a plain file named`/somedir/temporary.txt` is ignored, as is a directory named `/somedir/temp`.                 |
 | `*/*/temp*`    | Exclude files starting with name `temp` from any subdirectory that is two levels below the root directory. For example, the file `/somedir/subdir/temporary.txt` is ignored. |
 | `temp?`        | Exclude the files that match the pattern in the root directory. For example, the files `tempa`, `tempb` in the root directory are ignored.                                   |
-| `*.md `        | Exclude all markdown files in the root directory.                                                                                                                            |
-| `!LICENSE.md`  | Exception to the Markdown files exclusion is this file,  `LICENSE.md`, Include this file in the build.                                                                       |
-
-The placement of  `!` exception rules influences the matching algorithm; the
-last line of the `.dockerignore` that matches a particular file determines
-whether it is included or excluded. In the above example, the `LICENSE.md` file
-matches both the  `*.md` and `!LICENSE.md` rule. If you reverse the lines in the
-example:
-
-```
-    */temp*
-    */*/temp*
-    temp?
-    !LICENSE.md
-    *.md
-```
-
-The build would exclude `LICENSE.md` because the last `*.md` rule adds all
-Markdown files in the root directory back onto the ignore list. The
-`!LICENSE.md` rule has no effect because the subsequent `*.md` rule overrides
-it.
-
-You can even use the  `.dockerignore` file to ignore the `Dockerfile` and
-`.dockerignore` files. This is useful if you are copying files from the root of
-the build context into your new container but do not want to include the
-`Dockerfile` or `.dockerignore` files (e.g. `ADD . /someDir/`).
-
+| `*.md `        | Exclude all markdown files in the root directory...       |                                  
+| `!README*.md`  | ... except those matching `README*.md`. These markdown file *will* be included in the build...
```

Comment:
Actually, the ellipses do not indicate omissions but only that a single sentence spans more than one line, and different lines of the `.dockerignore` file correspond to different parts of that sentence.  Maybe it would be all right if I just removed them?

Point taken about the future tense.